doc/data-flow/README.md

```diff
@@ -0,0 +1,6 @@
+This is a sequence diagram for a typical cortx end-to-end data-flow.
+It shows processing of an S3 PUT request.
+
+![bb0](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/andriytk/cortx-motr/gen-data-flow-uml/doc/data-flow/s3-0.txt&fmt=svg)
```

Comment:
This URL "https://raw.githubusercontent.com/andriytk/cortx-motr/gen-data-flow-uml/doc/data-flow/s3-0.txt" should be replaced by
"https://raw.githubusercontent.com/Seagate/cortx-motr/gen-data-flow-uml/doc/data-flow/s3-0.txt"
?

Reply:
Yes, after the landing.

Reply:
Let's change it now.

Or we can commit the "s3-[0-2].txt" first and then commit this file with modified URL.
At that time, this file can be viewed properly.

Reply:
Done.